### PR TITLE
Specify boot version

### PR DESCRIPTION
**Pull request by adsteel**
With the current boot version this tutorial doesn't always work as expected.

### CODE REVIEW

File: doc/second-edition/tutorial-01.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ To install `java` follow the [instructions][3] for your operating
 system. To install `boot` follow the very easy instructions in the
 corresponding section of its [README][4].
 
+> Note: For this tutorial to work as expected, make sure you install boot version 2.4.2
+
 Test the installation by issuing the `boot -h` command at the
 terminal. Then submit the `boot -u` command to get the latest `boot`
```
Comment (680098) [inline doc/second-edition/tutorial-01.md]: Love

**Comment by magomimmo** [comment]
I'm on a tablet right now. The problem it's not about the boot release in se, but about the boot release set in the boot.properties. Tomorrow I'll be back from vacation an I'll fix the text ...

**Comment by adsteel** [comment]
@magomimmo Sounds good!

**Comment by magomimmo** [comment]
As said I updated the text with an Attention note suggesting to do as follows before to run the boot command:

```bash
cd /path/to/modern-cljs
BOOT_VERSION=2.4.2 boot -V > boot.properties
```

Cheers

**Pull request closed without merging**